ios/Info.plist

```diff
@@ -22,16 +20,18 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSMicrophoneUsageDescription</key>
+	<string>We need access to your microphone to let others hear you.</string>
 	<key>UIApplicationSceneManifest</key>
 	<dict>
 		<key>UIApplicationSupportsMultipleScenes</key>
-		<true/>
+		<false/>
 	</dict>
 	<key>UIApplicationSupportsIndirectInputEvents</key>
 	<true/>
 	<key>UILaunchScreen</key>
 	<dict/>
-	<key>UIMainStoryboardFile</key>
+	<key>UILaunchStoryboardName</key>
```

Comment:
So was this the cause that it didn't show the GUI initially?

Reply:
this, in combination with UIApplicationSupportsMultipleScenes (multi window).